_posts/2015-11-23-my-first-couple-of-months-at-codurance.md

```diff
@@ -0,0 +1,87 @@
+
+I also noticed that many things I learnt at such events and conferences wouldn't always be immediately recognised or accepted at the workplace. And moving to another work environment didn't always solve this problem fully. I found that I wasn't learning and applying what I wanted to, from my peers and very few were really in tuned with what the community was about. So one fine day I decided to take charge of my career and make a serious decision and take up the Apprenticeship program offered by [Codurance](http://codurance.com/) and go through the process.
+
+I was urged to go this way after being inspired by Sandro's book: [The Software Craftsman](http://www.amazon.co.uk/books/dp/0134052501), attending all the [SoCraTes UK](http://socratesuk.org/) conferences, and meeting with developers who valued and took pride of their work namely their craft. 
+
+
+### Where we are just now
+
+It's now been nearly two months since I have been working for [Codurance](http://codurance.com/), a formidable force. And so it's also about time that I share my experiences with my fellow mates and the community around me.
+
+During my first few weeks at [Codurance](http://codurance.com/), I have been busy learning various things that have been chalked out for becoming a craftsman.
+
+When working on a kata or learning a concept, we paired or did what is known as 'mob programming' along with other apprentices and craftspersons. And most of the time used the [pomodoro technique](https://en.wikipedia.org/wiki/Pomodoro_Technique). Time boxing our work in intervals is something done both in groups and working individually. We would have a lot of discussions and retrospectives after working on a problem or writing some code from scratch. 
+
+### Structure of my program
+
+We used an internal tool based on the concept of [Impact Mapping](http://www.impactmapping.org/). I soon got interested in it when I saw my colleague [Franzi](https://twitter.com/Singsalad) (who is now a craftswoman) had used it to plan out her Apprenticeship route. Such a tool helps map out our goals and the tasks we need to perform to achieve it. And this can differ from person-to-person, depending on what they want to work on (driven by the Apprentice).
+
+My mentor and other craftspersons reviewed them to get an idea of what I wanted to achieve for myself. And then its upto me to apply my own drive and perseverance to achieve the individual stories. I and my mentor meet and talk informally on a regular basis, many a times pairing on a kata or a project or on the white board trying to get my head around a concept.
+
+### Days in the life of an Apprentice
+
+I found the working hours quite flexible, remote working is also an optio as we can do given everything is on the cloud these days. Our co-founders are understanding and compassionate about our individual situations.
```

Comment:
Remote working is not necessary an option. It's an option when you are on the bench or if the client allows, if you are in a project. I don't want to send the wrong message to people that may want to join Codurance in the future. 

Remove "everything is on the cloud these days". Everyone knows that and the information is redundant.